templates/Sutil-Safe/Build.fs

```diff
@@ -39,10 +39,12 @@ module Helpers =
 module Paths =
   let deploy = Path.getFullName "deploy"
   let client = Path.getFullName "src/Client"
+  let output = Path.getFullName "src/Client/output"
   let server = Path.getFullName "src/Server"
   let shared = Path.getFullName "src/Shared"
 
   let clientTest = Path.getFullName "test/Client"
+  let clientTestPublish = Path.getFullName "test/Client/public"
```

Comment:
@Jlll1 shall we go with clientTestOutput here to keep things consistent?

Reply:
Yep, that would be great.